### PR TITLE
cal, chage: add Chinese translation

### PR DESCRIPTION
**Pull request by anderly011**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/linux/cal.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cal
+
+> 打印日历信息，高亮显示当前日期。
+> 更多信息：https://manned.org/cal.
```
Comment (navarroaxel) [inline pages.zh/linux/cal.md:4]:
```suggestion
> 更多信息：<https://manned.org/cal>.
```

File: pages.zh/linux/chage.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# chage
+
+> 更改用户账户和密码到期信息。
+> 更多信息：https://manned.org/chage.
```
Comment (navarroaxel) [inline pages.zh/linux/chage.md:4]:
```suggestion
> 更多信息：<https://manned.org/chage>.
```

File: pages.zh/linux/chage.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo chage -M -1 {{用户名}}`
+
+- 设置账户到期日期：
+
+`sudo chage -E {{YYYY-MM-DD}}`
+
+- 强制用户在下次登录时更改密码：
+
+`sudo chage -d 0`
```
Comment (navarroaxel) [inline pages.zh/linux/chage.md:24]: This should be fixed in the main page too.
```suggestion
`sudo chage -d {{0}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @anderly011, welcome and thanks for your submission. I left some comments.

**Review by navarroaxel — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages.zh/linux/chage.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# chage
+
+> 更改用户账户和密码到期信息。
+> 更多信息：<https://manned.org/chage>.
+
+- 列出用户的密码信息：
+
+`chage -l {{用户名}}`
+
+- 启用密码在10天内过期：
```
Comment (blueskyson) [inline pages.zh/linux/chage.md:10]: Almost done, thank you. Please place one space before/after English words and numbers.
```suggestion
- 启用密码在 10 天内过期：
```

**Review by blueskyson — APPROVED** [review]

**Comment by navarroaxel** [comment]
Congratulations @anderly011 :tada:

**Comment by anderly011** [comment]
Thanks all!

**Review by hmsjy2017 — COMMENTED** [review]
😀

**Pull request merged**